### PR TITLE
Update ui_screens.md

### PR DESCRIPTION
**Pull request by dsarkar**
(Closes https://github.com/corona-warn-app/cwa-documentation/issues/508)
image

### CODE REVIEW

File: ui_screens.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ Accessibility of the application was one of our top priorities when designing an
 
 ## Screens
 
+Here you can find screenshots of some of the features that the Corona-Warn-App offers in the current release: [Take a Look](https://www.coronawarn.app/en/screenshots/). The correspoinding German screenshot can be found here: [Machen Sie sich selbst ein Bild](https://www.coronawarn.app/de/screenshots/).
```
Comment (MikeMcC399) [inline ui_screens.md:17]: The correspoinding German screenshot can be found here
correspoinding => corresponding
screenshot => screenshots

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
I noted two typos.

@dsarkar 
Just out of interest, which editor do you use?
Can you integrate a spell check in the editor?

File: ui_screens.md (hunk 1 above)
Comment (dsarkar) [reply ui_screens.md:17]: ok thanks

**Comment by dsarkar** [comment]
@MikeMcC399 I use Visual Studio Code. I will check about a spell checker plugin. Thanks!

**Pull request merged**